app/controllers/bulk_update/recommendations_checks_controller.rb

```diff
@@ -6,7 +6,7 @@ class RecommendationsChecksController < ApplicationController
 
     def show
       @recommendations_upload ||= provider.recommendations_uploads.find(params[:recommendations_upload_id])
-      @awardable_rows_count = @recommendations_upload.awardable_rows.size
+      @awardable_rows_count = @recommendations_upload.awardable_rows.count
```

Comment:
Why do we need `count` here? `size` will fall back to `count` if the relation isn't already loaded.